Common/include/CConfig.hpp

```diff
@@ -1618,7 +1667,7 @@ class CConfig {
    * \brief Get the p-norm for heat-flux objective functions (adjoint problem).
    * \return Value of the heat flux p-norm
    */
-  su2double GetPnormHeat(void);
+  su2double GetPnormHeat(void) const { return pnorm_heat; }
```

Comment:
I dont believe this is used anymore.

Reply:
it seems to be used actually in CNEMONSSolver::Friction_Forces